editor/src/extension.js

```diff
@@ -182,10 +180,15 @@ const Exporter = () => {
 			content,
 		};
 
+		const { meta, ...filteredParams } = window.tiTpc.params;
+		console.log( { meta, filteredParams } );
+		console.log( tiTpc.params );
+		return;
```

Comment:
Not sure if this was meant for the commit?

Reply:
Thank you for that, I did not push the cleanup changes, I'm sorry for that. Should be good now.